### DISCUSSION TITLE
I need help creating a portfolio website

### DISCUSSION THREAD

**Discussion by Blueskies94** [community/community#32257] [Other Feature Feedback, Questions, & Ideas]
I see in GitHub pages that I built the site from the main branch and / (root). When I added files to the repository, it uses the index.html site instead of the one I created with all the pictures. 

https://github.com/Blueskies94/Blueskies94.github.io is my site. Under files, you'll see the index-real.html which is the site I've worked on. But instead, it hosts, index.html which is straight from html5. Let me know if you can find the issue. Thanks!

**Comment by radar-blip** [comment]
Can you just delete the index.html and rename index-real.html to index.html? Should be easy enough. 😕

**Reply by Blueskies94** [reply]
I just did. Unfortunately: https://blueskies94.github.io/ the website is still showing the old, unmodified interface like it was never changed.

**Reply by vinaymishra1201** [reply]
yes u can change

**Reply by Blueskies94** [reply]
> yes u can change

Ok, I just clicked on the link and it was been successfully updated! I know I'm a bit new to GitHub, but I was a little impatient and confused about the process. Thanks, I guess that's all and I'll remember the process in case things go wrong again!

**Reply by radar-blip** [reply]
Yeah, I thought it looked different now. Glad you got it fixed!
